code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -0,0 +1,118 @@
+/*
+Stabilized extracts:
+	Provides a passive buff to the holder.
+*/
+
+//To add: Create an effect in crossbreeding/_status_effects.dm with the name "/datum/status_effect/stabilized/[color]"
+//Status effect will automatically be applied while held, and lost on drop.
+
+/obj/item/slimecross/stabilized
+	name = "stabilized extract"
+	desc = "It seems inert, but anything it touches glows softly..."
+	effect = "stabilized"
+	icon_state = "stabilized"
+	var/datum/status_effect/linked_effect
+	var/mob/living/owner
+
+/obj/item/slimecross/stabilized/Initialize()
+	..()
+	START_PROCESSING(SSobj,src)
+
+/obj/item/slimecross/stabilized/Destroy()
+	STOP_PROCESSING(SSobj,src)
+	qdel(linked_effect)
+	return ..()
+
+/obj/item/slimecross/stabilized/process()
+	var/humanfound = null
+	if(ishuman(loc))
+		humanfound = loc
+	if(ishuman(loc.loc)) //Check if in backpack.
+		humanfound = (loc.loc)
+	if(!humanfound)
+		return
+	var/mob/living/carbon/human/H = humanfound
+	var/sanitizedcolour = replacetext(colour, " ", "")
+	var/effectpath = text2path("/datum/status_effect/stabilized/[sanitizedcolour]")
```

Comment:
Same thing as with the crossing, use type iteration please.

Reply:
Oh, yep, missed this!